README.md

```diff
@@ -188,149 +476,514 @@ Options:
 ```
 
-Or using the built JavaScript:
+**NOTE**: When using this option, `statementResults`, `clauseResults`, `evaluatedResource`, and `html` will no longer be present on the results object. If you need to access these, do not disable verbose calculation.
 
-```bash
-npm run build
-node build/cli.js [command] [options]
-```
+## Measure Logic Highlighting
 
-Optionally, you can install the `ts-node` utility globally to execute the TypeScript files directly instead of running the build script:
+`fqm-execution` can provide highlighted HTML to indicate which individual pieces of the measure logic CQL had "truthy" or "falsy" values. Generation of this HTML is enabled by default, and can be disabled by setting the `calculateHTML` calculation option to `false`.
 
-```bash
-npm install -g ts-node
-ts-node src/cli.ts [command] [options]
+The following example shows a scenario where the logic highlighting for a patient indicates that they are not in the numerator population of the measure due to the timing constraints of a FHIR `Procedure` resource that exists on their record.
+
+![Screenshot of Highlighted HTML Measure Logic](./static/logic-highlighting-example.png)
+
+In detailed results calculation, an HTML string will appear on each `detailedResults` object for a given patient:
+
+```typescript
+import { Calculator } from 'fqm-execution';
+
+// ...
+
+const { results } = await Calculator.calculate(measureBundle, patientBundles, {
+  calculateHTML: true /* true by default, but explicitly shown for demonstration */
+});
+
+// results
+/*    ^?
+      [
+          {
+            "patientId": "patient-1",
+            "detailedResults": [
+              {
+                "groupId": "group-1",
+                "html": ""<div><h2>Population Group: group-1</h2> ..."
+                ...
+              }
+              ...
+            ]
+            ...
+          }
+          ...
+     ]
+*/
+```
+
+In `MeasureReport` calculation, an HTML string will appear on the `.text.div` property of the returned FHIR `MeasureReport` for a given patient:
+
+```typescript
+import { Calculator } from 'fqm-execution';
+
+// ...
+
+const { results } = await Calculator.calculateMeasureReports(measureBundle, patientBundles, {
+  calculateHTML: true /* true by default, but explicitly shown for demonstration */,
+  reportType: 'individual' /* highlighting is only available for individual MeasureReport calculation */
+});
+
+// results
+/*    ^?
+        [
+          {
+            "id": "<some-random-uuid>",
+            "resourceType": "MeasureReport",
+            "text": {
+              "status": "generated",
+              "div": "<div><h2>Population Group: group-1</h2> ..."
+            }
+            ...
+          }
+          ...
+        ]
+ */
 ```
 
-### Debug Option
+**NOTE**: Measure logic highlighting is not available when calculation a population `MeasureReport` with `reportType = 'summary'`
```

Comment:
```suggestion
**NOTE**: Measure logic highlighting is not available when calculating a population `MeasureReport` with `reportType = 'summary'`
```

Reply:
These non-typo wrong word errors will be the end of me